packages/cli/templates/platform-app-tailwind/manifest.json

```diff
@@ -1,10 +1,13 @@
 {
-    "appId": "Test App",
+    "appId": "changeThisIdToMatchMarketplace",
```

Comment:
Could also use the `<>` as they do it for content blocks (catches the eye more easily):
`"appId": "<Your app ID>"`

Reply:
```suggestion
    "appId": "<Your app ID>",
```